### PR TITLE
Feature/accounts templates

### LINKED ISSUE #72: Create sign up / sign in page
Ensure that uses can register and login to Dipor.

Definition of done
---

* [ ] Accounts template package is added to project.
* [ ] Sign in / sign up routes have been configured.
* [ ] Determine any other useful configuration options.
* [ ] Sign in / sign up pages exists.
* [ ] Sign in / sign up use Bootstrap styles.
* [ ] Link in navigation menu exists.

### PR DESCRIPTION
**Pull request by brylie**
Closes #72

# Proposed changes
- add useraccounts packages
  - useraccounts:bootstrap
  - useraccounts:flow-router
- Add routing for accounts pages
- Add basic accounts configuration

@Digipalvelutehdas/dipor-team please review.

### CODE REVIEW

**Comment by brylie** [comment]
@Digipalvelutehdas/dipor-team this PR is ready for review.

**Comment by jykae** [comment]
I will check this.

**Comment by jykae** [comment]
Could you show the logged in user's email for example on the left side of the button in navbar?

loggedin

File: accounts/routes.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Sign in
+AccountsTemplates.configureRoute('signIn', {
+  layoutType: 'blaze',
+  name: 'signin',
+  path: '/login',
+  //template: 'myLogin',
```
Comment (jykae) [inline accounts/routes.js]: Remove unused commented lines.

File: accounts/routes.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  layoutTemplate: 'mainLayout',
+  contentRegion: 'main'
+});
+
+// Register
+AccountsTemplates.configureRoute('signUp', {
+  layoutType: 'blaze',
+  name: 'register',
+  path: '/register',
+  //template: 'myLogin',
```
Comment (jykae) [inline accounts/routes.js]: Remove unused commented lines.

File: accounts/accounts.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  //       signUp: "Register"
+  //   },
+  //   socialSignUp: "Register",
+  //   socialIcons: {
+  //       "meteor-developer": "fa fa-rocket"
+  //   },
+  //   title: {
+  //       forgotPwd: "Recover Your Password"
+  //   },
+  // },
```
Comment (jykae) [inline accounts/accounts.js]: If you don't have plans for the commented lines here, please remove.

**Comment by jykae** [comment]
Well done. Mostly cleaning and that small usability enhancement.

**Comment by brylie** [comment]
I have cleaned up the file. I am hesitant to add the user email to the navigation menu, as it is not clear what purpose it would serve.

Perhaps we can consider displaying the user email on their profile page.

**Comment by brylie** [comment]
@jykae I have resolved conflicts. Please review/merge.

**Comment by bajiat** [comment]
@jykae Reassigned this to @elnzv. You have the RESTIVUS package to work on.

**Pull request merged**